lib/src/test/kotlin/graphql/nadel/validation/NadelFieldValidationTest.kt

```diff
@@ -676,5 +676,30 @@ class NadelFieldValidationTest : DescribeSpec({
 
             assert(error.message == "The overall field Query.fieldA defines argument id which does not exist in service test field Query.fieldA")
         }
+
+        it("fails if schema contains all fields as hidden") {
+            val fixture = NadelValidationTestFixture(
+                overallSchema = mapOf(
+                    "test" to """
+                type Query {
+                    echo: String @hidden
+                }
+            """.trimIndent(),
+                ),
+                underlyingSchema = mapOf(
+                    "test" to """
+                type Query {
+                    echo: String
+                }
+            """.trimIndent(),
+                ),
```

Comment:
should we have the opposite test - where there is 2 field with only 1 hidden

Reply:
We should and I have added it, Thanks @bbakerman !